drivers/SmartThings/zwave-switch/profiles/fibaro-double-switch.yml

```diff
@@ -14,52 +14,42 @@ components:
       version: 1
     categories:
     - name: Switch
-  - id: switch1
-    capabilities:
-    - id: switch
-      version: 1
-    - id: powerMeter
-      version: 1
-    - id: energyMeter
-      version: 1
-    categories:
-    - name: Switch
   - id: button1
     capabilities:
-    - id: button
-      version: 1
+      - id: button
+        version: 1
     categories:
-    - name: RemoteController
+      - name: RemoteController
   - id: button2
     capabilities:
-    - id: button
-      version: 1
+      - id: button
+        version: 1
     categories:
-    - name: RemoteController
+      - name: RemoteController
   - id: button3
     capabilities:
-    - id: button
-      version: 1
+      - id: button
+        version: 1
     categories:
-    - name: RemoteController
+      - name: RemoteController
   - id: button4
     capabilities:
-    - id: button
-      version: 1
+      - id: button
+        version: 1
     categories:
-    - name: RemoteController
+      - name: RemoteController
   - id: button5
     capabilities:
-    - id: button
-      version: 1
+      - id: button
+        version: 1
     categories:
-    - name: RemoteController
+      - name: RemoteController
   - id: button6
     capabilities:
-    - id: button
-      version: 1
+      - id: button
+        version: 1
     categories:
-    - name: RemoteController
+      - name: RemoteController
```

Comment:
can you revert these whitespace changes?

Reply:
> can you revert these whitespace changes?

done